src/bmffimage.cpp

```diff
@@ -184,7 +185,7 @@ long BmffImage::boxHandler(std::ostream& out /* = std::cout*/, Exiv2::PrintStruc
   if (bTrace) {
     bLF = true;
     out << indent(depth) << "Exiv2::BmffImage::boxHandler: " << toAscii(box_type)
-        << Internal::stringFormat(" %8ld->%lu ", address, box_length);
+        << Internal::stringFormat(" %8" PRIi32 "->%" PRIu64 " ", address, box_length);
```

Comment:
'address' is actually a long and not int32_t, so could/should leave it as %ld

Reply:
yeah ... I initially assumed that it was a `uint32_t` (by mistake) and I saw the LGTM error. Then I tried with `int32_t` but this is not the same as `long` ... In the [cinttypes](https://en.cppreference.com/w/cpp/header/cinttypes) doc, it does not seem to be anything to be used with the `long` type. I am surprised by this ...

I will leave it as `%8ld` for the moment.